scripts/prepublish

```diff
@@ -11,6 +11,16 @@ if ! npm outdated --long; then
   fi
 fi
 
+source 'node_modules/sanctuary-scripts/functions'
+
+pkg="$(get repo-name)"
+files="$(get source-files)"
+
+sed --in-place "s/$pkg@0.0.0/$pkg@$VERSION/" "$files"
+sed --in-place "s/$pkg@$PREVIOUS_VERSION/$pkg@$VERSION/" "$files"
```

Comment:
The `--in-place` flag is not supported on macOS; one must use `-i ''` on that platform. It may not be possible to write a portable script that uses this sed feature. Portability may not be important.

Strictly speaking the first pattern should use `0[.]0[.]0` rather than `0.0.0`.

`"$files"` only works correctly because __.config__ specifies only a single source file. I suggest using `"${files[@]}"` in conjunction with [`files=($(get source-files))`][1].


[1]: https://github.com/sanctuary-js/sanctuary-scripts/blob/v4.0.0/bin/lint#L10-L11

Reply:
Thanks, David. The portability problem I'm not going to solve now, but I noted it down in https://github.com/fluture-js/fluture-template/issues/10 

The other suggestions are great and I'll add them upstream before pulling them down and updating this PR.

Reply:
> Strictly speaking the first pattern should use `0[.]0[.]0` rather than `0.0.0`.

Hm, I guess the same is true for `$PREVIOUS_VERSION`, which also contains periods. I'd have to regex-escape that to be truly robust, or tell sed somehow that we want a literal match.